lib/kafka_rest_client.js

```diff
@@ -76,8 +78,29 @@ function KafkaRestClient(options, callback) {
     self.scheduleTopicMapRefresh();
     self.statsd = (options.statsd && typeof options.statsd.increment === 'function') ? options.statsd : null;
     self.metricsPrefix = 'kafka_rest_proxy.rest-client.' + os.hostname() + '.';
+
+    if ('localAgentPort' in options) {
+        self.localAgentPort = options.localAgentPort;
+    } else {
+        self.localAgentPort = defaultLocalAgentPort;
+    }
```

Comment:
There is a concise way like  this:
self.localAgentPort = options.localAgentPort || defaultLocalAgentPort;